python/health-check/src/health_check/config/templates/grafana_dashboard/supportconfig_with_logs.template.json

```diff
@@ -1380,13 +1343,13 @@
     ]
   },
   "time": {
-    "from": "",
-    "to": ""
+    "from": "now-30d",
+    "to": "now"
```

Comment:
Is this intentional change? We leave this empty because health-check tool sets this based on the passed-in parameters